### PR TITLE
home and product pages added

### PR DESCRIPTION
**Pull request by Sandesh9834**
image

image

### CODE REVIEW

**Review by kushalsourav — APPROVED** [review]
UI looks good , good to merge
decrease the size of badge over icons

File: Styles/home.css [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+    .image-card 
+    {
+      margin: 3.2rem 1rem;
+    }
+
+    .template 
+    {
+      position: absolute;
+      height: 31.25rem;
+      width: 89%;
```
Comment (Vivek-jpg) [inline Styles/home.css:30]: use proper calculation

File: Styles/product.css [hunk 2]
```diff
@@ -0,0 +1,160 @@
...
+      height: 100%;
+      overflow-y: auto;
+      width: 100%;
+      display: flex;
+      background-color: var(--white-color);
+}
+
+.filter-sidebar 
+{
+      height: 700px;
```
Comment (Vivek-jpg) [inline Styles/product.css:12]: use rem in place of px

File: Styles/product.css [hunk 3]
```diff
@@ -0,0 +1,160 @@
...
+
+.card-container {
+      width: 12.5rem;
+      background-color: var(--white-color);
+      border: 0.5px solid var(--light-gray-colo);
+      text-align: center;
+      position: relative;
+  }
+
+  .card-with-badge {
```
Comment (Vivek-jpg) [inline Styles/product.css:115]: this is good practice

File: Styles/product.css [hunk 4]
```diff
@@ -0,0 +1,160 @@
...
+
+img {
+  height: 12.5rem;
+  width: 12.5rem;
+  margin-bottom: 0;
+  padding: 0;
+}
+
+.description {
+  background-color: rgba(128, 128, 128, 0.363);
```
Comment (Vivek-jpg) [inline Styles/product.css:132]: col variables

File: Styles/product.css [hunk 5]
```diff
@@ -0,0 +1,160 @@
...
+div 
+{
+  display: block;
+}
+
+p {
+  display: block;
+  margin-block-start: 1em;
+  margin-block-end: 1em;
+  margin-inline-start: 0px;
```
Comment (Vivek-jpg) [inline Styles/product.css:158]: why you use margin 0  declare on top part in css

File: pages/home.html [hunk 6]
```diff
@@ -0,0 +1,80 @@
...
+        <a href="#" class="nav-heading">Sports Cart</a>
+        <input class="search-bar" placeholder="Search" />
+      </div>
+      <div class="right-nav flex position">
+        <button class="btn round-button"><a href="login.html">Login</a></button>
+        <button class="btn round-button"><a href="signup.html">Signup</a></button>
+      <button  class="btn-badge badge-lg ">
+        <a href="wishlist.html">
+        <i class="fas fa-heart fa-2x"></i>
+        </a>
```
Comment (Vivek-jpg) [inline pages/home.html:24]: naming is goood

File: pages/home.html [hunk 7]
```diff
@@ -0,0 +1,80 @@
...
+        <img class="shoe" src="sneakers-shoes-adidas-shoes.jpg"/>
+      </div>
+      <div class="image-card">
+        <img class="shoe" src="sneakers-shoes-adidas-shoes.jpg"/>
+      </div>
+      <div class="image-card">
+        <img class="shoe" src="sneakers-shoes-adidas-shoes.jpg"/>
+      </div>
+      <div class="image-card">
+        <img class="shoe" src="sneakers-shoes-adidas-shoes.jpg"/>
```
Comment (Vivek-jpg) [inline pages/home.html:46]: this BAM convention is good

File: pages/product.html [hunk 8]
```diff
@@ -0,0 +1,167 @@
...
+      <h1 class="showing-heading">Showing all products</h1>
+      <br>
+      <div class="product-flex">
+        <div class="card-container">
+          <h4 class="card-with-badge"><i class="fas fa-heart"></i></h4>
+          <img class="card-image" src="sneakers-shoes-adidas-shoes.jpg" alt="Adidas Shoe" />
+          <div class="description">
+            <p class="card-heading">Adidas Shoe</p>
+            <p class="price">₹3000</p>
+            <button class="add-cart-btn">Add to Cart</button>
```
Comment (Vivek-jpg) [inline pages/product.html:99]: rest is good

**Review by Vivek-jpg — APPROVED** [review]
can be merge

File: Styles/home.css [hunk 9]
```diff
@@ -0,0 +1,93 @@
+.shoe {
+      position: relative;
+      width: 18.75rem;
+      height: 12.5rem;
```
Comment (tarunsankhla) [inline Styles/home.css:4]: Try avoiding decimal in rem,it will get confusing on responsive on different screen

File: Styles/home.css [hunk 10]
```diff
@@ -0,0 +1,93 @@
...
+      top: 6.25rem;
+      text-align: center;
+      width: 18.75rem;
+      background-color: var(--white-color);
+    }
+
+    .hero 
+    {
+      margin: 0px 2.2rem;
+    }
```
Comment (tarunsankhla) [inline Styles/home.css:19]: No need to add 0px just 0will also work

File: Styles/product.css [hunk 11]
```diff
@@ -0,0 +1,160 @@
...
+      padding: 1rem 2rem;
+      font-weight: bolder;
+      font: 1.2em sans-serif;
+}
+    
+.range 
+{
+      display: flex;
+      justify-content: space-around;
+      font: 1em sans-serif;
```
Comment (tarunsankhla) [inline Styles/product.css:49]: You have used sans-serif font most of the time initialize in body won't need to declare everywhere

File: Styles/product.css [hunk 12]
```diff
@@ -0,0 +1,160 @@
...
+      margin-left: 2rem;
+      margin-top: 1rem;
+      width: 80%;
+}
+    
+.filter-cat span 
+{
+      font: 1.5em sans-serif;
+      padding: 1rem 2rem;
+
```
Comment (tarunsankhla) [inline Styles/product.css:64]: Extra spacing

File: pages/product.html [hunk 13]
```diff
@@ -0,0 +1,167 @@
...
+            <a href="cart.html">
+              <i class="fas fa-shopping-cart fa-2x"></i>
+            </a>
+            <span class="icon-button-badge flex center icon-xl">10</span>
+          </button>
+      </div>
+    </nav>
+
+</body>
+<div class="main-body-sec">
```
Comment (tarunsankhla) [inline pages/product.html:40]: Add comments

**Review by tarunsankhla — APPROVED** [review]
Overall classname and code is great, just minor changes

**Comment by AagmanSuri** [comment]
code looks clean , good to merge

**Pull request merged**